bin/bench-tx/src/main.rs

```diff
@@ -68,10 +68,10 @@ pub fn benchmark_default_tx() -> anyhow::Result<TransactionMeasurements> {
             Account::mock(ACCOUNT_ID_REGULAR_PUBLIC_ACCOUNT_UPDATABLE_CODE, IncrNonceAuthComponent);
 
         let input_note_1 =
-            create_p2any_note(ACCOUNT_ID_SENDER.try_into().unwrap(), &[FungibleAsset::mock(100)]);
+            create_p2any_note(ACCOUNT_ID_SENDER.try_into().unwrap(), [FungibleAsset::mock(100)]);
```

Comment:
nit: These can be
```suggestion
            create_p2any_note(ACCOUNT_ID_SENDER.try_into()?, [FungibleAsset::mock(100)]);
```
right? I'm not sure they are worth changing at all but it comes up pretty often